### PR TITLE
(QA-2357) cleanup setup.py

### PR DESCRIPTION
**Pull request by jpinsonault**
Some general cleanup to get ready for pypi release.

* Renamed the script to `vmpooler_client_app.py` to avoid import conflicts
* Update the gitignore to remove .egg-info folder
* Use `setuptools` instead of `distutils` to support building wheels

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from setuptools import setup
```
Comment (cowofevil) [inline setup.py]: There is a potential that ```setuptools``` might not be available on a system. I would suggest putting in this wrapper to allow fallback:

```
try:
  from setuptools import setup
except ImportError:
  from distutils.core import setup
```

**Comment by cowofevil** [comment]
Also you should bump the version number to 3 since we completely changed the name of the script.

File: setup.py (hunk 1 above)
Comment (jpinsonault) [inline setup.py]: Good point. I was thinking it's only a dev dependency but someone might for some inexplicable reason run setup.py manually. We're only using setuptools to be able to build the wheels

**Comment by jpinsonault** [comment]
Version bumped, made import better

**Pull request merged**